src/OpenSEE/Scripts/TSX/openSEE.tsx

```diff
@@ -308,7 +308,7 @@ class OpenSEEHome extends React.Component<OpenSee.IOpenSeeProps, OpenSee.iOpenSe
                     
                     <div style={{ padding: '0', height: "calc(100% - 62px)", overflowY: 'auto' }}>
                         {plotData[this.props.eventID] != undefined ?
-                            <div className="card" style={{borderLeft: 0, borderRight: 0}}>
+                            <div className="card" style={{ borderLeft: 0, borderRight: 0, borderBottomColor: 'transparent' }}>
```

Comment:
Technically this does resolve the issue. however, that's not really a good solution since it leaves the div there. I'd recommend you figure out a way to not include that div if it is only empty.